contrib/packaging/rpm/cfg/Dockerfile

```diff
@@ -1,24 +1,13 @@
-FROM fedora:24
+FROM fedora:25
 
-MAINTAINER "Andre Martins <andre@cilium.io>"
+LABEL "Maintainer: Marcin Skarbek <marcin@skarbek.name>"
 
-RUN dnf install -y gettext curl gcc fedora-packager fedora-review && \
-cd /tmp && \
-curl -Sslk -o go1.7.4.linux-amd64.tar.gz \
```

Comment:
Dockerfiles no, at leas not for builds used in main repo. Deckerfiles are part of the new initiative - Fedora Modularity but that doesn't change anything. `fedpkg` run as `mockbuild` will use Golang from repo which means Golang form repo in Dockerfile.